src/main/java/utils/ReadJsonAsString.java

```diff
@@ -0,0 +1,22 @@
+package utils;
+
+import org.apache.commons.text.StringSubstitutor;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+
+public class ReadJsonAsString {
+
+    public String readFileAsString(String file)throws Exception
+    {
+        return new String(Files.readAllBytes(Paths.get(file)));
+    }
+
+    public String parseJsonTemplate(String filePath, Map<String, Object> values) throws Exception {
+        String jsonTxt = new ReadJsonAsString().readFileAsString(filePath);
```

Comment:
new ReadJsonAsString() you do not need to call new here, the method `readFileAsString(String file)` belongs to this class